packages/devtools-network-console/src/reducers/websocket/index.ts

```diff
@@ -79,9 +86,15 @@ export default function reduceWebsocket(collection: WS_State = DEFAULT_WS_STATE,
         if (!state) {
             state = DEFAULT_WS_CONNECTION;
         }
+        // TODO: add disconnected reason/error to content
         state = {
             ...state,
-            connected: false
+            connected: false,
+            messages: state.messages.add({
+                direction: 'status',
+                time: 0,
```

Comment:
Guess I'll need to add disconnect-time from the host ;)

Reply:
possibly. We aren't currently displaying it in status. I'll make it optional in the Message type, and we can decide if we want to add it later when we handle server disconnect reason

Reply:
put time as an optional param on websocket messages for now.